spec/system/users_show_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+RSpec.describe 'Users#show', type: :feature, js: true do
+  before(:all) do
+    User.destroy_all
+    Post.destroy_all
+    Comment.destroy_all
+    @lilly = User.create!(name: 'Lilly', photo: 'https://portraits.ai', bio: 'Teacher from Poland.', posts_counter: 0)
+    @post = Post.create!(title: 'First Post', text: 'This is my first post', author: @lilly, comments_counter: 0,
+                         likes_counter: 0)
+    @tom = User.create!(name: 'Tom', photo: 'https://portraits.ai', bio: 'Teacher from Mexico.', posts_counter: 0)
+    @users = User.all
+  end
+  # I can see the user's profile picture.
+  it 'displays the user profile picture' do
+    visit '/users'
+    click_on 'Lilly'
+    expect(page).to have_css("img[src*='#{@lilly.photo}']")
+  end
+  # I can see the user's username.
+  it 'displays the user username' do
+    visit '/users'
+    click_on 'Lilly'
+    expect(page).to have_content(@lilly.name.to_s)
+  end
+  # I can see the number of posts the user has written.
+  it 'displays the number of posts the user has written' do
+    visit '/users'
+    click_on 'Lilly'
+    expect(page).to have_content('1')
+  end
+  # I can see the user's bio.
+  it 'displays the user bio' do
+    visit '/users'
+    click_on 'Lilly'
+    expect(page).to have_content('Teacher from Poland.')
+  end
+  # I can see the user's first 3 posts.
+  it 'displays the user first 3 posts' do
+    visit '/users'
+    click_on 'Lilly'
+    expect(page).to have_content('First Post')
+  end
```

Comment:
- [ ] @Team Great work adding tests for the users' show page, however, on this test, you were supposed to test if the `three_recent_posts` method is working correctly, when its presented with more than 3 posts it displays the latest 3 posts, kindly consider reconfiguring this test.